tools/compiletest/src/read2.rs

```diff
@@ -6,77 +6,29 @@
 // FIXME: This is a complete copy of `cargo/src/cargo/util/read2.rs`
```

Comment:
I looked at the latest Cargo util directory, https://github.com/rust-lang/cargo/tree/master/src/cargo/util, and there doesn't seem to be any file named `read2.rs` anymore. Is this copied code outdated? Should we consider updating it?

Reply:
I removed the comment.